code/game/turfs/flooring/flooring.dm

```diff
@@ -115,6 +115,8 @@ var/list/flooring_types
 	var/list/movable_atom_whitelist = list()
 	var/list/movable_atom_blacklist = list()
 
+	var/do_season = FALSE	//VOREStation Addition
```

Comment:
This var could be named a bit more obvious too, like 'seasonal' or 'check_season'.